### ISSUE TITLE
Add config setting to not log creation of taxomony terms

### ISSUE THREAD

**Issue by mjordan** [mjordan/islandora_workbench#558]
During `create` tasks with `allow_adding_terms` set to `true`, a lot of static is added to the workbench log, e.g.:

```
06-Feb-23 22:00:15 - INFO - Node for "The role of assistive devices and technologies in residents' quality of life and staff care practices in long-term care facilities: A narrative synthesis of the literature (record 7163)" created at https://summit.sfu.ca/node/35792.
06-Feb-23 22:00:22 - INFO - Summit URL in the TRS for node 35792 (ETDID etd22163) updated (TRS PATCH response code: 200).
06-Feb-23 22:00:22 - INFO - TRS user record for knmurray (ETDID etd22163) disabled (TRS PATCH response code: 200).
06-Feb-23 22:00:26 - INFO - Media for /home/zlocal/summit_data/tmp/etd22163/etd22163.pdf created.
06-Feb-23 22:00:26 - INFO - Term 78153 ("Particle accelerator") added to vocabulary "tags".
06-Feb-23 22:00:27 - INFO - Term 78154 ("Superconducting radio frequency cavity") added to vocabulary "tags".
06-Feb-23 22:00:27 - INFO - Term 78155 ("Logitudinal vibration") added to vocabulary "tags".
06-Feb-23 22:00:28 - INFO - Term 78156 ("Flexural dynamic") added to vocabulary "tags".
06-Feb-23 22:00:28 - INFO - Term "Kalman filter" (term ID 16782) already exists in vocabulary "tags".
06-Feb-23 22:00:29 - INFO - Term 78157 ("LQG") added to vocabulary "tags".
06-Feb-23 22:00:29 - INFO - Term 78158 ("Keikha, Mahsa") added to vocabulary "person".
06-Feb-23 22:00:29 - INFO - Term "Moallem, Mehrdad" (term ID 27898) already exists in vocabulary "person".
06-Feb-23 22:00:30 - INFO - Term "Applied Sciences: School of Mechatronic Systems Engineering" (term ID 34288) already exists in vocabulary "sfu_department".
06-Feb-23 22:00:31 - INFO - Node for "Active vibration modeling and control in radio frequency (RF) cavities (record 7270)" created at https://summit.sfu.ca/node/35793.
06-Feb-23 22:00:31 - INFO - Summit URL in the TRS for node 35793 (ETDID etd22270) updated (TRS PATCH response code: 200).
06-Feb-23 22:00:31 - INFO - TRS user record for mkeikha (ETDID etd22270) disabled (TRS PATCH response code: 200).
06-Feb-23 22:00:34 - INFO - Media for /home/zlocal/summit_data/tmp/etd22270/etd22270.pdf created.
06-Feb-23 22:00:35 - INFO - Term 78159 ("Adult ADHD") added to vocabulary "tags".
06-Feb-23 22:00:35 - INFO - Term 78160 ("Social camouflaging") added to vocabulary "tags".
06-Feb-23 22:00:36 - INFO - Term 78161 ("Internalized stigma") added to vocabulary "tags".
06-Feb-23 22:00:36 - INFO - Term "Social anxiety" (term ID 35337) already exists in vocabulary "tags".
06-Feb-23 22:00:36 - INFO - Term "Generalized anxiety" (term ID 33693) already exists in vocabulary "tags".
06-Feb-23 22:00:37 - INFO - Term "Depression" (term ID 16159) already exists in vocabulary "tags".
06-Feb-23 22:00:37 - INFO - Term 78162 ("Mylett, Marisa") added to vocabulary "person".
06-Feb-23 22:00:37 - INFO - Term "Iarocci, Grace" (term ID 30835) already exists in vocabulary "person".
06-Feb-23 22:00:37 - INFO - Term "Arts & Social Sciences: Department of Psychology" (term ID 27672) already exists in vocabulary "sfu_department".
```

Users should be able to add a setting to their configs to not log creation of new terms.

**Comment by mjordan** [comment]
Adding `log_term_creation: false` to you "create" and "update" task configs will remove logging the creation of new terms. `--check` still reports that they will be created (in other words, this new setting has no effect during `--check`. Also has no effect in "create_terms" tasks.

**Comment by mjordan** [comment]
Closed with 51348d00498baa4efa72afa0daf78044a8e3a458.
